services/ui-src/src/components/reports/StandardReportPage.test.tsx

```diff
@@ -1,82 +1,34 @@
 import { render, screen } from "@testing-library/react";
-import userEvent from "@testing-library/user-event";
 import { axe } from "jest-axe";
 // components
 import { ReportContext, StandardReportPage } from "components";
 // utils
-import { useUser } from "utils";
 import {
-  mockAdminUser,
   mockForm,
   mockReportContext,
   mockStandardReportPageJson,
-  mockStateUser,
   RouterWrappedComponent,
 } from "utils/testing/setupJest";
 
-const mockUseNavigate = jest.fn();
-jest.mock("react-router-dom", () => ({
-  useNavigate: () => mockUseNavigate,
-  useLocation: jest.fn(() => ({
-    pathname: "/mock",
-  })),
-}));
```

Comment:
Request: Let's keep these and add a test so we have coverage for StandardReportPage line 29 (the next navigation on the successful form check and subsequent "submission", which is just the navigation method.

Reply:
yeah we're fine here.